test/integration/targets/azure_rm_acr/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/azure
```

Comment:
Let's add `posix/ci/cloud/group2/azure` alias to run as part of CI. See more at [here](https://github.com/ansible/ansible/blob/devel/test/utils/shippable/cloud.sh#L10)

Reply:
yes done

Reply:
#32481reported. temp remove it to pass ci.